### ISSUE TITLE
WLAN monitor issue

### ISSUE THREAD

**Issue by relimS** [maxnet/berryboot#695]
I think BerryBoot is causing some firmware issues. I could not get Kali Linux to use wlan0 in monitor mode (wlan0mon) as it thinks monitor mode does not exist which should not be the case because it works fine before I started using BerryBoot. I am running Raspberry Pi Kali Linux in July 2019 for the image. I have only changed the filesystem (to SquashFS) and (I believe) nothing else
![Screenshot 2021-03-08 15_47_42](https://user-images.githubusercontent.com/66686057/110325398-e18cc880-8049-11eb-85b1-60aa9bed79e4.png)
![Screenshot 2021-03-08 15_43_06](https://user-images.githubusercontent.com/66686057/110325671-33355300-804a-11eb-8171-c4e65833d732.png)

**Comment by relimS** [comment]
It work fine with PINN so I guess I will use PINN instead

**Comment by Nsomnia** [comment]
Yep. One of the reasons I keep a pinn sd or usb around, thst and the Xbian or.whwyebe it's called kodi image for rpi4 on pinn. What you lose is versatility since BB you can add and remove images as desired so pinn I just add 8 project spaces and 500mb data partition as well as kali dietpi kodi x2 raspi 64 lite and a couple others. I can't decide which one is pefect to tgis day so both it is. Whichever you use less, probably pinn with kali, put on a cheap brick n motor store sale sd card.

With a usb adapter it works though and a.resltek 8188 based can be had for 5 bucks on ebay locally
